### ISSUE TITLE
Allow to turn off diagnostics

### ISSUE THREAD

**Issue by krstp** [zed-industries/community#1649]
### Check for existing issues

- [X] Completed

### Describe the feature

Sometimes I work on non-standard file. To enable coloring I set a given syntax (be it C, python, whatever). Syntax coloring really helps to read the code/text/data. However, when syntax is set, ZED feels a duty to underline and mark all incorrectly formatted/syntaxed lines/objects/items. It is a little bit disturbing if the whole file is out of given syntax and everything is generally underlines as "error".

I would like to have an option to be able to turn of diagnostics on the tab of choice.

### If applicable, add mockups / screenshots to help present your vision of the feature

![diag](https://github.com/zed-industries/community/assets/4711498/6ae21cb2-d0dc-4fee-979d-ba8b4ac9d27d)
Above see a sample with all the entries underline.  I would like to be able to select syntax of choice and be able to turn of diagnostics on such file.

**Comment by krstp** [comment]
Yeah, this would be much desired. Maybe I am mistaken, but seems this should be a relatively easy feature to implement... of course, I am making assumption, but would be great to see it. At time, those wiggle lines really interrupt the work flow ;) Thanks for all the hard work ZED team does!

**Comment by krstp** [comment]
A side note, this has nothing to do with inline diagnostics, such as in #1857. What I would like to see is just the ability to turn off wiggles on particular pane all together ;) Thanks!

**Comment by JosephTLyons** [comment]
Hey @krstp, I'm actually a little unsure of how you were able to get errors to show up in a csv file in general. I created a CSV file and applied a language (C) to get syntax highlighting, but I don't see any error reporting on my end for that file.  Can you tell me what I'm missing?
